llvm/test/ObjectYAML/GOFF/GOFF-Header-End.yaml

```diff
@@ -0,0 +1,18 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
+
+# Verify that GOFF Header is correct.
```

Comment:
Nit: our newer tests in this area tend to use `##` for comments to help  them stand out from lit and FileCheck lines.

On the note of comments, it might be useful to include comments saying what each line represents.